packages/commerce-sdk-react/src/hooks/ShopperOrders/mutation.ts

```diff
@@ -16,48 +17,40 @@ export const ShopperOrdersMutations = {
      */
-    CreateOrder: 'createOrder',
-    /**
-   * Adds a payment instrument to an order. 
-
-Details:
-
-The payment instrument is added with the provided details. The payment method must be applicable for the order see GET
-/baskets/\{basketId\}/payment-methods, if the payment method is 'CREDIT_CARD' a paymentCard must be specified in the request.
-   * @see {@link https://developer.salesforce.com/docs/commerce/commerce-api/references/shopper-orders?meta=createPaymentInstrumentForOrder} for more information about the API endpoint.
-   * @see {@link https://salesforcecommercecloud.github.io/commerce-sdk-isomorphic/classes/shopperorders.shopperorders-1.html#createpaymentinstrumentfororder} for more information on the parameters and returned data type.
-   */
-    CreatePaymentInstrumentForOrder: 'createPaymentInstrumentForOrder',
-    /**
-     * Removes a payment instrument of an order.
-     * @see {@link https://developer.salesforce.com/docs/commerce/commerce-api/references/shopper-orders?meta=removePaymentInstrumentFromOrder} for more information about the API endpoint.
-     * @see {@link https://salesforcecommercecloud.github.io/commerce-sdk-isomorphic/classes/shopperorders.shopperorders-1.html#removepaymentinstrumentfromorder} for more information on the parameters and returned data type.
-     */
-    RemovePaymentInstrumentFromOrder: 'removePaymentInstrumentFromOrder',
-    /**
-   * Updates a payment instrument of an order.
-
-Details:
-
-The payment instrument is updated with the provided details. The payment method must be applicable for the
-order see GET /baskets/\{basketId\}/payment-methods, if the payment method is 'CREDIT_CARD' a
-paymentCard must be specified in the request.
-   * @see {@link https://developer.salesforce.com/docs/commerce/commerce-api/references/shopper-orders?meta=updatePaymentInstrumentForOrder} for more information about the API endpoint.
-   * @see {@link https://salesforcecommercecloud.github.io/commerce-sdk-isomorphic/classes/shopperorders.shopperorders-1.html#updatepaymentinstrumentfororder} for more information on the parameters and returned data type.
-   */
-    UpdatePaymentInstrumentForOrder: 'updatePaymentInstrumentForOrder'
+    CreateOrder: 'createOrder'
 } as const
 
-type ShopperOrdersMutationType = typeof ShopperOrdersMutations[keyof typeof ShopperOrdersMutations]
+export type ShopperOrdersMutationType = typeof ShopperOrdersMutations[keyof typeof ShopperOrdersMutations]
+
+export const queryKeysMatrix = {
+    createOrder: (
+        data: DataType<Client['createOrder']>,
+        params: Argument<Client['createOrder']>
+    ): QueryKeysMatrixElement => {
+        const customerId = data?.customerInfo?.customerId
+        return {
+            update: [['/orders', {orderNo: data.orderNo}]],
+            invalidate: [['/customers', customerId, '/baskets', {customerId}]]
```

Comment:
Don't need object at the end for invalidate because of react query's fuzzy matching